Cargo.toml

```diff
@@ -17,8 +17,11 @@ thiserror = "1.0.34"
 tokio = { version = "1.21.1", features = ["rt-multi-thread", "time"] }
 uniffi = "0.20.0"
 
+esplora-client = { git = "https://github.com/bitcoindevkit/rust-esplora-client" }
```

Comment:
Specifying the async feature would safe us compilation time and create a smaller binary
`
esplora-client = { git = "https://github.com/bitcoindevkit/rust-esplora-client", default-features = false, features = ["async"] }`

Reply:
Good point, thanks.